### PR TITLE
Clean code a bit and fix typos in code documentation

### PR DESCRIPTION
**Pull request by ariasuni**

### CODE REVIEW

File: build.rs [hunk 1]
```diff
@@ -50,9 +50,10 @@ fn write_statics() -> IOResult<()> {
     use std::io::Write;
     use std::path::PathBuf;
 
-    let ver = match is_development_version() {
-        true   => format!("exa v{} ({} built on {})", cargo_version(), git_hash(), build_date()),
-        false  => format!("exa v{}", cargo_version()),
```
Comment (ogham) [inline build.rs:55]: I actually really like matching on a boolean, but whenever I show code using it to anyone, I get weird looks. So I suppose this is for the best. _wistful sigh_

**Comment by ogham** [comment]
This is really great, thank you.

File: build.rs (hunk 1 above)
Comment (ariasuni) [reply build.rs:55]: Oh sorry I realize now that it’s rude due to the name of my PR and the absence of description. As you may have seen in the commit messages, it’s a change suggested by clippy that I tend to follow except if it’s really uglier.

Comment (ogham) [reply build.rs:55]: Don't worry, I didn't take it as rude at all.

**Pull request merged**